src/uu/wc/src/wc.rs

```diff
@@ -146,6 +146,10 @@ fn wc(files: Vec<String>, settings: &Settings) -> StdResult<(), i32> {
     let mut results = vec![];
     let mut max_width: usize = 0;
 
+    // we do not need to decode the byte stream if we're only counting bytes/newlines
+    let decode_chars = settings.show_chars
+        || settings.show_words || settings.show_max_line_length;
```

Comment:
```suggestion
    let decode_chars = settings.show_chars || settings.show_words || settings.show_max_line_length;
```
to make rustfmt happy